app/api/auth/create_access_token.go

```diff
@@ -73,6 +97,15 @@ import (
 //       redirect_uri:
 //         type: string
 //         description: OAuth2 redirection URI
+//       use_cookie:
+//         type: boolean
+//         description: If true, set a cookie instead of returning the OAuth2 code in the data
+//       cookie_secure:
+//         type: boolean
+//         description: If true, set the cookie with the `Secure` attribute
+//       cookie_same_site:
+//         type: boolean
+//         description: If true, set the cookie with the `SameSite`='Strict' attribute value (instead of `SameSite`='None')
 // responses:
 //   "201":
 //     description: "Created. Success response with the new access token"
```

Comment:
I supposed the struc of the resp must be updated as well to allow `access_token` not to be given.

Reply:
fixed